mantidproject/mantid
# Remove duplicate WISH system test files

**Description of work.**

Different versions of the following files existed in both the Testing\Data\SystemTest and Testing\SystemTests\tests\framework\reference:

WISH40503-5_6no_absorb_raw.nxs.md5
WISH40503-5_6raw.nxs.md5
WishAnalysis.nxs.md5

The files in the Testing\Data folder seemed to always get found first by Mantid during the system tests. This was a bit confusing because it was possible to update the ones in the "reference" folder without having any effect. 

The files should only exist in the Testing\SystemTests\tests\framework\reference because they're used in the validate method of the WISH system tests. The changes here delete the files from Testing\Data and bring the files in the reference folder up to date

**To test:**

No code changes - just changes to reference files. So just review files changes look sensible and check Jenkins tests pass

Fixes #30908.

*This does not require release notes* because **no functional change**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
